### PR TITLE
Add board and hny provider

### PR DESCRIPTION
**Pull request by sgsharma**
Add a k8s metrics board example and add a provider file

### CODE REVIEW

**Review by puckpuck — APPROVED** [review]

File: terraform/k8s_board.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  filter {
+    column = "k8s.resource.type"
+    op     = "="
+    value = "node"
+  }
+
+  breakdowns = ["k8s.node.name"]
+  orders     = [
```
Comment (jharley) [inline terraform/k8s_board.tf:19]: This should be:

```hcl
  order {
      column = "metrics.cpu.usage"
      op         = "AVG"
      order    = "descending"
    }
```

**Review by jharley — CHANGES_REQUESTED** [review]
Lookin' good! I'd recommend running `terraform fmt` generally as it makes things pretty and is a syntax check

File: terraform/k8s_board.tf (hunk 1 above)
Comment (sgsharma) [reply terraform/k8s_board.tf:19]: Gotcha! I'll add that to the GH actions flow as well 👍

**Review by jharley — APPROVED** [review]
Nice! ✨

**Pull request merged**